packages/core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -110,7 +110,8 @@ const Dropdown: VibeComponent<DropdownComponentProps, HTMLElement> & {
       popupsContainerSelector,
       filterOption,
       menuPosition = Dropdown.menuPositions.ABSOLUTE,
-      "data-testid": dataTestId
+      "data-testid": dataTestId,
+      withGroupDivider = false
```

Comment:
do we have any way to put it in the supplied options array instead of as a prop?

Reply:
We wanted to keep it similar to the combobox api that has `withCategoriesDivider`. Also if it is in the options array will have to send the prop in each group separately.

Reply:
oh you're right
good call 💪🏼